Library/Formula/glib.rb

```diff
@@ -45,6 +45,11 @@ class Glib < Formula
     url "https://gist.githubusercontent.com/jacknagel/9835034/raw/282d36efc126272f3e73206c9865013f52d67cd8/gio.patch"
     sha256 "d285c70cfd3434394a1c77c92a8d2bad540c954aad21e8bb83777482c26aab9a"
   end
+  
+  patch do 
+    url "https://git.gnome.org/browse/glib/patch/?id=4a292721bcf2943bfc05c6a1c859992f28e3efec"
+    sha256 "6a5b330c58e42c31fb025fb312740a7dc510c7d7d1205e88c9b13918da6da9f8"
+  end
```

Comment:
It's already committed upstream. See #38874. Just needs a comment added  explaining what it does, etc. ` https://mail.gnome.org/archives/commits-list/2015-March/msg10287.html ` is probably a good reference to point at.